README.md

```diff
@@ -1,54 +1,95 @@
+
+<p align="left">
+  <img src="/docs/wavefront_java_tracing_agent.png">
+</p> 
+
+The Wavefront Java Tracing Agent includes:
+* The [Java OpenTracing SpecialAgent](https://github.com/opentracing-contrib/java-specialagent), which automatically instruments Java applications with traces.
+* The Wavefront OpenTracing Bundle that is implemented in this repository, which sends the tracing data to Wavefront for observability.
+ 
+## Prerequisites
+
+*  Java 7 or above.
+* [Download](https://github.com/opentracing-contrib/java-specialagent#2111-stable) the latest version of the Java OpenTracing SpecialAgent to your application's directory.
+
+## Configuring the Parameters
+
+Follow the steps given below:
+
+1. You can configure the [Tracer parameters](#Parameters) using any of the following methods.
+  * Create a file named `tracer.properties` in the application's directory and use the following template to configure the properties.
+
+    This file is used to configure your Wavefront Tracer instance for reporting and to configure tags specific to the application.
+  
+    Example:
+    ```properties
+    # Required application tags
+    wf.application=myApp
+    wf.service=myService
+
+    # Optional application tags
+    wf.cluster=us-west
+    wf.shard=primary
+
+    # Reporting through direct ingestion
+    wf.reportingMechanism=direct
+    wf.server=<replace-with-wavefront-url>
+    wf.token=<replace-with-wavefront-api-token>
+
+    # Reporting with a Wavefront proxy
+    #wf.reportingMechanism=proxy
+    #wf.proxyHost=<replace-with-wavefront-proxy-hostname>
+    #wf.proxyMetricsPort=2878
+    #wf.proxyDistributionsPort=2878
+    #wf.proxyTracingPort=30000
+    ```
+
+  * Configure the parameters by overriding the existing parameters using System properties.
+
+    Example:
+    ```bash
+    java -cp:$MYCLASSPATH:wavefront-opentracing-bundle-java.jar \
+        -Dwf.service=MyOtherService \
+        com.mycompany.MyOtherService
+    ```
+
+  * Configure the Tracer parameters via YAML files. You need two YAML files:
+    * One to 
+  [configure application tags](https://github.com/wavefrontHQ/wavefront-jersey-sdk-java#1-configure-application-tags).
+    * Another to [configure Wavefront reporting](https://github.com/wavefrontHQ/wavefront-jersey-sdk-java#2-configure-wavefront-reporting). 
+
+    The paths to these YAML files need be specified in `tracer.properties` or as System properties:
+
+    Example: Define the paths to the YAML files via System porperties.
+    ```bash
+    java -cp:$MYCLASSPATH:wavefront-opentracing-bundle-java.jar \
+        -Dwf.applicationTagsYamlFile=application-tags.yaml \
+        -Dwf.reportingConfigYamlFile=wf-reporting-config.yaml \
+        com.mycompany.MyService
+    ```
+
+    **Note**: *The parameters configured via `tracer.properties` or System properties override the parameters configured via YAML files*.
+2.  Add `-Dsa.tracer=wavefront` to send traces to Wavefront.<br/>
```

Comment:
Can update to "[Attach](https://github.com/opentracing-contrib/java-specialagent#22-usage) the Java OpenTracing SpecialAgent to your application, adding `-Dsa.tracer=wavefront` to send traces to Wavefront."